src/pull.go

```diff
@@ -22,6 +22,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"sort"
```

Comment:
I decided not to generate message inside of function as it is different for pull and push case, is that ok?